acceptance_tests/cypress/integration/lotse_flow.spec.js

```diff
@@ -284,7 +284,10 @@ context('Acceptance tests', () => {
                 cy.get('#person_a_town').type(taxReturnData.personA.town)
                 cy.get(submitBtnSelector).click()
 
-                cy.get('label[for=person_a_has_disabilityno]').click()
+                cy.get('label[for=person_a_has_disabilityyes]').click()
+                cy.get(submitBtnSelector).click()
+
+                cy.get('label[for=person_a_requests_pauschbetragyes]').click()
```

Comment:
As this tests without deductions, I think we do not need to set disabilities in this test case (the other two are good though)

Reply:
:+1: